### PR TITLE
Added content in the tutorials file

### PR DESCRIPTION
**Pull request by jefftheprogrammer**
I made some changes to the tutorials file, mostly in _index.md, basic-movement.md, python-whirlwind-tour.md and simulator.md.

### CODE REVIEW

File: content/tutorials/_index.md [hunk 1]
```diff
@@ -3,7 +3,41 @@ title: Tutorials
...
 
 * [Basic Movement](basic-movement) - explains how to use the kit to operate a motor
 * [Python: A Whirlwind Tour](python-whirlwind-tour) - a guide to using Python, aimed towards those who haven't used it before
+* [A basic guide to using the simulator](simulator) - a guide to using the we-bot simulator
+
+## Installation
+
+### Prerequisites
+
+You need to download and install Webots (the download is around 1.5GB). This is the platform we run our simulation in. Whilst we recommend version “R2021a” of Webots, and will be running competition simulations using it, “R2020b-rev1” should still work.
```
Comment (sedders123) [inline content/tutorials/_index.md:16]:
```suggestion
You need to download and install Webots R2021b (the download is around 0.5GB). This is the platform we run our simulation in.
```

File: content/tutorials/_index.md [hunk 2]
```diff
@@ -3,7 +3,41 @@ title: Tutorials
...
+You will also need Python installed.
+
+| Platform | Supported Python Version|
+| -------- | ----------------------- |
+| Windows | 3.9 (64 bit) |
+| macOS | 3.9 |
+| Linux | 3.9 |
+
+In the competition, python 3.9 will be used.
+There are a small number of [extenal libraries](https://github.com/sourcebots/docs/blob/sp2021/content/api/_index.md#included-libraries) which will be available to your robot code during the competition. For local development you will need to install these yourself.
```
Comment (sedders123) [inline content/tutorials/_index.md:29]: I think this is the correct way to link? (@RealOrangeOne please shout if I'm wrong!)
```suggestion
There are a small number of [extenal libraries](/content/api#included-libraries) which will be available to your robot code during the competition. For local development you will need to install these yourself.
```

File: content/tutorials/_index.md [hunk 3]
```diff
@@ -3,7 +3,40 @@ title: Tutorials
...
 ---
 
-This part of the docs contains some practical guides for getting starting with both Python and our kit.
+This part of the docs contains some practical guides for getting starting with both Python and the simulator.
 
 * [Basic Movement](basic-movement) - explains how to use the kit to operate a motor
 * [Python: A Whirlwind Tour](python-whirlwind-tour) - a guide to using Python, aimed towards those who haven't used it before
+* [A basic guide to using the simulator](simulator) - a guide to using the we-bot simulator
+
+## Installation
```
Comment (RealOrangeOne) [inline content/tutorials/_index.md:12]: Should this section not be on the simulator page?

File: content/tutorials/_index.md [hunk 4]
```diff
@@ -3,7 +3,40 @@ title: Tutorials
...
+
+### Prerequisites
+
+You need to download and install Webots R2021b (the download is around 0.5 GB). This is the platform we run our simulation in.
```
Comment (RealOrangeOne) [inline content/tutorials/_index.md:16]: They don't actually need webots downloaded to use the simulator, just vscode. Perhaps linking out to that and saying they'll get a link to join liveshare at the event?

File: content/tutorials/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sedders123) [reply content/tutorials/_index.md]: d7bd180

File: content/tutorials/_index.md [hunk 5]
```diff
@@ -3,7 +3,8 @@ title: Tutorials
 weight: 1
 ---
 
-This part of the docs contains some practical guides for getting starting with both Python and our kit.
+This part of the docs contains some practical guides for getting starting with both Python and the simulator.
 
 * [Basic Movement](basic-movement) - explains how to use the kit to operate a motor
 * [Python: A Whirlwind Tour](python-whirlwind-tour) - a guide to using Python, aimed towards those who haven't used it before
+* [A basic guide to using the simulator](simulator) - a guide to using the we-bot simulator
```
Comment (RealOrangeOne) [inline content/tutorials/_index.md:10]:
```suggestion
* [A basic guide to using the simulator](simulator) - a guide to using the Webots simulator
```

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**